solidity/contracts/Gravity.sol

```diff
@@ -521,11 +521,19 @@ contract Gravity is ReentrancyGuard {
 		}
 	}
 
+	function sendToCronos(
+		address _tokenContract,
+		address _destination,
+		uint256 _amount
+	) public nonReentrant {
+		sendToCosmos(_tokenContract, bytes32(uint256(uint160(_destination))), _amount);
+	}
+
 	function sendToCosmos(
 		address _tokenContract,
 		bytes32 _destination,
 		uint256 _amount
-	) public nonReentrant {
+	) private {
```

Comment:
Can we open both function to be public so users can choose the one they're comfortable to?

Reply:
But we need to remove `nonReentrant`, does that make it less safe to use? no strong opinion on it.

Reply:
Or can we keep the `nonReentrant` on both `sendToCosmos` and `sendToCronos` somehow?

Reply:
ahhh I just realised the "confusing" part is people want to send to `Cronos` but instead they need to call `sendToCosmos`? 😂 

If that's the case I think you're right, we can keep it this way

Reply:
The most confusing part is in `sendToCosmos`, the `bytes32` type, when you send a 20bytes address to it, by default the client pad it with zeroes on the right side, which results in the wrong address on Cronos side, you need to explicitly pad it on the left side.

Reply:
I notice there's a new method `sendToCosmosEth` now, maybe we also need a variant that accept recipient in `address` type?